packages/api-server/src/modules/config.ts

```diff
@@ -18,13 +18,15 @@ export class ConfigController {
     return ConfigController.singleton;
   }
 
-  async get({ ws }: WorkspaceRequest): Promise<RespV2<ConfigGetPayload>> {
+  async get({
```

Comment:
We should probably try to get rid of this controller altogether at some point (later), I don't see what it's necessary.